### DISCUSSION TITLE
Tables are inefficient in solidjs

### DISCUSSION THREAD

**Discussion by solid-component** [TanStack/table#5297] [Q&A]
Modifying tables in solidjs results in full rendering, not fine-grained updates, if that's the case, then why don't I just use react?

**Comment by goatfryed** [comment]
Do you make sure to follow solidjs's rules for incremental ui updates?
Can you provide an example to reproduce the issue, maybe a codesandbox?

I'm not a maintainer myself, but I don't think your question is in a state where you could get help.
